content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-job-stories/diagram.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: blocks/image_content_block
+image: content-strategy/user-story-map.svg
+link: content-strategy/user-story-map.svg
+reverse: false
+stackPosition: top
+imageAlt: Example user stories and tasks of a content specialist
+level: 2
+display: 3
+long_description: long-description.md
+---
+
+### [3]Step 5: Map your stories
```

Comment:
This section heading should be added to the YAML at the top of the file under the `link` param as `title: Step 5: ...[etc]`
The`level` and `display` params in the YAML need to be adjusted to '3' and '2' respectively.

Reply:
Steve fixed.